make/src/main.rs

```diff
@@ -87,22 +137,51 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         env::set_current_dir(dir)?;
     }
 
-    let parsed = parse_makefile(makefile.as_ref()).unwrap_or_else(|err| {
-        eprintln!("make: {}", err);
-        process::exit(err.into());
-    });
-    let config = Config {
+    let mut config = Config {
         ignore,
         dry_run,
         silent,
         touch,
+        env_macros,
+        quit,
+        keep_going,
+        clear,
+        terminate,
+        ..Default::default()
```

Comment:
I don't think this is a good idea to use default here, as the initial idea was to supply exhaustivity and enforce the explicit use of all the arguments

Reply:
Yes, but there is also a field `rules` which must be default initialized in order to have default rules defined in standard

Reply:
So use the default implementation for the needed field (`rules`)